mis_builder/views/mis_report_instance.xml

```diff
@@ -192,6 +192,7 @@
                                 <field name="no_auto_expand_accounts" />
                                 <field name="display_columns_description" />
                                 <field name="hide_analytic_filters" />
+                                <field name="inverse_detail" />
```

Comment:
Place this field next to `no_auto_expand_accounts` as they are closely related.
Also this field should be invisible if `no_auto_expand_accounts` is checked.